.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: 'ci'
```

Comment:
Only add these when we start to have the corresponding code. Otherwise the checks will fail (as they are failing now)